src/components/symptom-shark/view/SymptomDetailView/SymptomDetailView.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import SymptomSharkSymptomDetailView, { SymptomDetailViewProps } from "./SymptomDetailView";
+import { demoSymptoms } from "../../helpers/demo-data";
+
+export default { title: "SymptomShark/View/SymptomDetailView", component: SymptomSharkSymptomDetailView, parameters: { layout: 'fullscreen' } };
+let render = (args: SymptomDetailViewProps) => <SymptomSharkSymptomDetailView {...args} previewState="default" />
```

Comment:
May want to wrap this one with `<Layout colorScheme="auto">` as well?

Reply:
Actually, I guess these views already have layouts within them.  Maybe it actually needs removed from the `LogEditEntryViews` stories.